Migration.md

```diff
@@ -19,6 +19,8 @@ release will remove the deprecated code.
       * CMake `-config` files
       * Paths that depend on the project name
 
+  1. Python library `ignition.math` namespaces should be replaced with `gz.math7`.
```

Comment:
```suggestion
  1. Python library imports such `import ignition.math` and `from ignition import math` should be replaced with `import gz.math7` and `from gz import math7`. Note the change from `ignition` to `gz` and the addition of the major version number as a suffix to the package name.
```

Reply:
should we recommend `from gz import math7 as math`?

Reply:
> should we recommend from gz import math7 as math?

ouch, sorry I missed this one. I'm fine about using that recommendation, we can modify the Changelog but specially use the documentation and examples to do it.

Reply:
I didn't suggest it very quickly, but yes we can consider it for documentation